### PR TITLE
[12.0][IMP] hr_contract_currency: creation with currency and default company

### PR DESCRIPTION
**Pull request by alexey-pelykh**
Backport of improvement from @astirpe

### CODE REVIEW

File: hr_contract_currency/tests/test_hr_contract_currency.py [hunk 1]
```diff
@@ -34,3 +35,25 @@ def test_1(self):
...
+            'company_id': my_company.id,
+        })
+
+        self.assertEqual(
+            contract.currency_id,
+            self.env.ref("base.CHF")
+        )
+        self.assertNotEqual(
+            contract.currency_id,
+            self.env.company.currency_id
```
Comment (astirpe) [inline hr_contract_currency/tests/test_hr_contract_currency.py]:
```suggestion
            self.env.user.company_id.currency_id
```

**Review by astirpe — CHANGES_REQUESTED** [review]

**Review by astirpe — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Ugh, again these timezone hiccups :|

**Pull request merged**